### ISSUE TITLE
Tracked tensor in TensorForce agent

### ISSUE THREAD

**Issue by devantheryl** [tensorforce/tensorforce#859]
Hello,

I have a problem to track the parameters of the agent. 

At the beginning when I launch the agent with the following settings :

`agent = Agent.create(
            agent='ddqn',
            states = environment.states(),
            actions = environment.actions(),
            max_episode_timesteps = environment.max_episode_timesteps(),
            memory=memory,
            batch_size = batch_size,
            network = network,
            update_frequency = update_frequency,
            learning_rate = dict(type = 'linear', unit = 'episodes', num_steps = int(num_episode*0.5), initial_value = 0.01, final_value = learning_rate),
            #huber_loss = huber_loss,
            horizon = horizon,
            discount = discount,
            target_update_weight = target_update_weight ,
            target_sync_frequency  = target_sync_frequency,
            exploration = dict(type = 'linear', unit = 'episodes', num_steps = int(num_episode*0.8), initial_value = epsilon, final_value = epsilon_min),
            config = dict(seed = 1),
            tracking = 'all',
            parallel_interactions  = 1,
        )`

**Comment by AlexKuhnle** [comment]
Hi @devantheryl,

When you say they are not visible, do you mean that they still show the "old" values? If so, that may be a bug in how the tracked_tensors() feature is implemented (i.e. I'm pretty sure the new values will be used, they are just not returned).

**Comment by devantheryl** [comment]
Hi @AlexKuhnle , thank's for the quick answer :)

No, I mean the values are not present at all.

new agent created with tracked_tensor = 'all' :

![image](https://user-images.githubusercontent.com/55431595/158772354-d4477fbd-d3a1-49b5-80d8-5ce9ee4fdc99.png)


agent loaded and exploration + learning rate modified as mentionned above : 

![image](https://user-images.githubusercontent.com/55431595/158773222-380815bd-23f2-4c91-ac07-754956b79005.png)

as you can see, the two modified values aren't present anymore in the tracked values.

**Comment by AlexKuhnle** [comment]
Ah, right, so it's precisely the modified values which are missing... Okay, need to look into that.
